### DISCUSSION TITLE
AttributeError: module 'types' has no attribute 'UnionType'. Did you mean: 'FunctionType'?

### DISCUSSION THREAD

**Discussion by lolio-dev** [fastapi/fastapi#6242] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [x] I commit to help with one of those options 👆

### Example Code

```python
This is the app where I have the error 'https://github.com/lolio-dev/Shoppingify-server/tree/main/app' but I also have the error with the most basic app: 

from fastapi import FastAPI

app = FastAPI()

@app.get('/')
def root():
    return {'hello': 'word'}    
```
```


### Description

With a very simple and basic application I have the error `AttributeError: module 'types' has no attribute 'UnionType'. Did you mean: 'FunctionType'?`. The error happens when I import fastapi 
![ksnip_20220625-201701](https://user-images.githubusercontent.com/58036293/175785782-70e59ea9-ba3e-49d5-a88d-576ab7c0e997.png)
I have only install `pip install fastapi[all]` 

### Operating System

Linux

### Operating System Details

_No response_

### FastAPI Version

0.78.0

### Python Version

3.10

### Additional Context

_No response_

**Comment by JarroVGIT** [comment, marked as answer]
I am not sure if it would help but I would recommend to update your Python installation to a stable version. You are currently on beta 4, so might be because of that ?

**Comment by lolio-dev** [comment]
Oh it was the solution, it was a risky installation of python3.10 and I did not seen that it was in beta version. Now with the last version all is ok. Thanks you so much !!
